client/controllers/incidentReports/incidentTable.coffee

```diff
@@ -0,0 +1,35 @@
+Template.incidentTable.onCreated ->
+  @rowsSelected = new ReactiveVar(false)
+
+selectedIncidents = (event, instance) ->
+  # return all the selected incident-ids
+  $('table.incident-table tr.active td.count').map(() ->
+    $(this).data("incident-id")
+  ).get()
+
+changeIncidentStatus = (status) ->
+  incidentIds = selectedIncidents()
+  $.each incidentIds, (index, incidentId) ->
+    Meteor.call 'getIncidentReport', incidentId, (error, incident) ->
```

Comment:
@jgoley - I was thinking it wouldn't be needed initially too, but when I tried to call `editIncidentReport` it was throwing validation errors because nothing else for the IR was set.  I can play around some more to see if there is a way to set only the field I care about without specifying all the other fields.